Octokit/Models/Response/User.cs

```diff
@@ -23,6 +27,13 @@ public User(string avatarUrl, string bio, string blog, int collaborators, string
         /// </summary>
         public bool SiteAdmin { get; protected set; }
 
+        /// <summary>
+        /// LDAP Binding (GitHub Enterprise only)
+        /// </summary>
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Dn")]
+        [SuppressMessage("Microsoft.Naming", "CA1709:IdentifiersShouldBeCasedCorrectly", MessageId = "Dn")]
+        public string LdapDn { get; protected set; }
```

Comment:
This could just be my inexperience with the way these things are working.  

The Microsoft Code Analysis stuff wanted me to put LdapDN but when you do that, the json serialiser tried to look for field called ```ldap_d_n``` so in order to get the API field ```ldap_dn``` deserialized I had to use ```ldapDn``` on the c# side, and suppress the warnings about "Dn" not being an allowed 2 letter word.  

I didnt realise we could "control" what 2 letter words are allowed.  Does that apply to the Microsoft Code Analysis stuff or the API field formatting stuff, or both?

I also wasnt sure if I could set what the "api name" of a given property is (ive seen it done on Enum entries, but then also seen mention that that method didnt work on both serialisation and deserialization).  If that is indeed possible then I could rename the field LdapDinstinguishedName and just "tag" it as ```ldap_dn``` for API purposes.  If this is posible, can you point me at a good example?